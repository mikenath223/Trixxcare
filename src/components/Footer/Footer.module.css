.sideFoot {
  height: 200px;
  width: 100%;
  text-align: center;
  position: relative;
  font-size: 25px;
  color: rgba(0, 0, 0, 0.9);
}

.sideFoot button {
  border: none;
  outline: none;
  font-size: 22px;
  font-weight: 600;
  background-color: #fff;
  color: inherit;
  margin-bottom: 30px;
  cursor: pointer;
}

.sideFoot p:last-child {
  position: absolute;
  bottom: 10px;
  display: flex;
  width: 100%;
  justify-content: space-around;
  cursor: pointer;
}

.sideFoot p:nth-child(2) {
  margin-top: -20px;
}