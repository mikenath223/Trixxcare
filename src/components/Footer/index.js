import React from 'react';
import PropTypes from 'prop-types';
import style from './Footer.module.css';

const Footer = ({ handleLogout, logged, handleConfirmed }) => (
  <footer className={style.sideFoot}>
    {logged ? <button type="button" onClick={handleLogout} tabIndex="-3">Logout</button>
      : <button type="button" onClick={handleConfirmed} tabIndex="-4">Sign In</button>}
    <p>
      {' '}
      <a href="https://github.com/mikenath223/trixxcare/issues" target="_blank" rel="noopener noreferrer" tabIndex="-5">Help</a>
    </p>
    <p>
      <img src="https://img.icons8.com/android/24/000000/twitter.png" alt="" />
      <img src="https://img.icons8.com/android/24/000000/facebook-new.png" alt="" />
      <img src="https://img.icons8.com/android/24/000000/google-plus.png" alt="" />
      <img src="https://img.icons8.com/material/24/000000/vimeo.png" alt="" />
      <img src="https://img.icons8.com/metro/24/000000/pinterest.png" alt="" />
    </p>
  </footer>
);

Footer.propTypes = {
  handleLogout: PropTypes.func.isRequired,
  logged: PropTypes.string.isRequired,
  handleConfirmed: PropTypes.func.isRequired,
};

export default Footer;
