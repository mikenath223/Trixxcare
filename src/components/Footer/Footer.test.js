import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from './index';

describe('Footer', function () {
  afterEach(cleanup);

  beforeEach(() => {
    this.props = {
      logged: '',
      handleConfirmed: jest.fn(),
      handleLogout: jest.fn(),
    };
  });

  it('should match snapshot', () => {
    const { container } = render(<Footer {...this.props} />);

    expect(container).toMatchSnapshot();
  });
});
