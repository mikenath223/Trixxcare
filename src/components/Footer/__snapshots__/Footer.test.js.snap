// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div>
  <footer
    class="sideFoot"
  >
    <button
      tabindex="-4"
      type="button"
    >
      Sign In
    </button>
    <p>
       
      <a
        href="https://github.com/mikenath223/trixxcare/issues"
        rel="noopener noreferrer"
        tabindex="-5"
        target="_blank"
      >
        Help
      </a>
    </p>
    <p>
      <img
        alt=""
        src="https://img.icons8.com/android/24/000000/twitter.png"
      />
      <img
        alt=""
        src="https://img.icons8.com/android/24/000000/facebook-new.png"
      />
      <img
        alt=""
        src="https://img.icons8.com/android/24/000000/google-plus.png"
      />
      <img
        alt=""
        src="https://img.icons8.com/material/24/000000/vimeo.png"
      />
      <img
        alt=""
        src="https://img.icons8.com/metro/24/000000/pinterest.png"
      />
    </p>
  </footer>
</div>
`;
